CONTRIBUTING.md

```diff
@@ -105,6 +105,7 @@ The following is the list of supported scopes:
 - **schematics**
 - **store**
 - **store-devtools**
+- **data**
```

Comment:
Let's move it to the top to keep the alphabetical order

Reply:
done in d8219b8.